tests/utils.py

```diff
@@ -97,8 +97,11 @@ def create_node_population(filepath, pop_name, circuit=None, node_sets=None):
     if node_sets is not None:
         circuit.node_sets = node_sets
     node_pop = NodeStorage(config, circuit).population(pop_name)
+
     if isinstance(circuit.nodes, dict):
         circuit.nodes[node_pop.name] = node_pop
```

Comment:
Is this `if` branch actual? `circuit.nodes` now always Nodes, no?

Reply:
Good eye, you are right !
I kept it for the edge tests but I changed stuffs in the edge tests and can remove it.